webapp-src/locales/nl/translations.json

```diff
@@ -42,6 +42,7 @@
     "error-value-expected": "Waarde vereist",
     "grant-title": "{{- client}} vraagt de volgende toegangsrechten",
     "grant": "Toegang toestaan",
+    "grant-none-available": "No grants have been requested",
```

Comment:
No need to include this one in the Dutch translation, if a i18n isn't found in the selected language, the fallback language (i.e. english) will be used

Reply:
@babelouest while the line is technically redundant, I inserted it as a trigger for Dutch-speaking contributors to fill-in the missing translation. Your call entirely, of course...

Reply:
I know someone who could help us ;)